scholia/app/templates/base.html

```diff
@@ -56,7 +56,12 @@
 	     $( '#details' ).append( '<a href="' + fullTextUrl + '">' + fullTextUrl + '</a>. ');
 	 }
 	 catch(e) {}
-	 
+
+	 try {
+	     var twitterAccount = item.claims.P2002[0].mainsnak.datavalue.value;
+	     $( '#details' ).append( '<a href="https://twitter.com/' + twitterAccount + '">@' + twitterAccount + '</a> ');
```

Comment:
I wonder if we might want a escape here, - just in case? I see that the above fullTextUrl does not have one...

Reply:
Escape of what? I think the Twitter account name is meant to give valid URLs.